src/main/java/com/mraof/minestuck/computer/theme/ComputerTheme.java

```diff
@@ -0,0 +1,73 @@
+package com.mraof.minestuck.computer.theme;
+
+import com.google.gson.*;
+import com.mojang.logging.LogUtils;
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.JsonOps;
+import com.mraof.minestuck.Minestuck;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.GsonHelper;
+import net.minecraft.util.valueproviders.ConstantInt;
+import org.slf4j.Logger;
+
+import java.lang.reflect.Type;
+
+/**
+ * This class is used to define json files that will be used to determine what wallpaper and text color appears on in-game computers.
+ */
+public class ComputerTheme
+{
+	private static final Logger LOGGER = LogUtils.getLogger();
+	
+	public static final int DEFAULT_TEXT_COLOR = 0x404040;
+	public static final ResourceLocation DEFAULT_TEXTURE_PATH = new ResourceLocation(Minestuck.MOD_ID, "textures/gui/theme/default.png");
+	
+	private final ResourceLocation texturePath;
+	private final ConstantInt textColor;
```

Comment:
I'm not sure there's much of a point to using `ConstantInt` in favor of a regular `int` unless you're abstracting over `IntProvider`. I suggest simplifying this by replacing it with a regular int.